### PR TITLE
Add Achintya’s weekly meeting notes

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
Signed-off-by: Achintya Rao <achintya@achintyarao.in>

### CODE REVIEW

**Comment by malvikasharan** [comment]
Thanks for this PR @RaoOfPhysics. Thank you so much putting in thoughts and care into the foundational work of ASG community papers. I hope the discussions from last weeks have been helpful in addressing some of the questions that you have listed in your recent notes. Since you took notes offline, I have not tried to summarise our discussion under each section. DO come back and add your notes if useful for the futrue reference. However, what is important IMHO is for us to block a day and work on some of the strategy documents to help us convey the vision for the project and community.

Have a lovely weekend! Talk to you next week.

**Review by malvikasharan — APPROVED** [review]

**Comment by RaoOfPhysics** [comment]
Thanks, @malvikasharan! Much appreciated. :)

Also, I would love for you to show me how you edit a PR directly!

File: project-management/1-1-meetings/Achintya-Rao/weekly-meetings.md [hunk 1]
```diff
@@ -10,17 +10,21 @@
 
 ## 2022-01-11
 
-- Check-in ritual
-    - Reflected on finding my voice as CM over the break
-    - Grateful to KW for help with goals
+**Check-in ritual**
+- Reflected on finding my voice as CM over the break
+    - I am glad to see that you could make some time on reflecting this. Looking forward to discussing this and see where I can support this iterative process.
```
Comment (RaoOfPhysics) [inline project-management/1-1-meetings/Achintya-Rao/weekly-meetings.md:15]: <3

File: project-management/1-1-meetings/Achintya-Rao/weekly-meetings.md [hunk 2]
```diff
@@ -10,17 +10,21 @@
...
 
 ### Project-specific reflection
 
 - Need to meet individual members of ASG to understand the community
+    - Who among the core stakeholders you have not yet managed to meet? 
 - Can identify connections once I speak to individual members
-- “Nobody knows how this community-building looks like.”
-  - What are the needs? What are the connections that exist between teams?
+- “Nobody knows how this community-building looks like.” What are the needs? What are the connections that exist between teams?
+  - I think many of us have the vision for ASG community and overview of how building them would look like. I hope discussion earlier in the week was useful, howeever, we should block a day to work on sketching out some concrete strategy and timeline to share them with others. This will help in communicating what we want to achieve and where we might be misaligned - which would be a place to further feedback and iteration, and especially for engagement.
```
Comment (RaoOfPhysics) [inline project-management/1-1-meetings/Achintya-Rao/weekly-meetings.md:25]: This will be helpful. :)

**Pull request merged**